packages/edit-site/src/components/editor-canvas-container/index.js

```diff
@@ -136,8 +136,7 @@ function EditorCanvasContainer( {
 					>
 						{ shouldShowCloseButton && (
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you go to site editor --> styles --> revisions, there is close button.

| Before | After |
| :-------- | :------- |
| <img width="361" alt="editor-canvas-close-before" src="https://github.com/user-attachments/assets/56f6e52d-021f-46d8-b32e-9910e9db0b10"> | <img width="361" alt="Editor-anvas-close-after" src="https://github.com/user-attachments/assets/c7a67ba5-f0fc-4168-a521-ad4392cb4fc8"> |